libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushMessaging.java

```diff
@@ -383,12 +381,13 @@ static long getBackoff(Context context, UserAccount account) {
      * @param backoff Backoff time to be used.
      * @param account User account.
      */
+    @SuppressWarnings("unused")
     static void setBackoff(Context context, long backoff, UserAccount account) {
         final SharedPreferences prefs = context.getSharedPreferences(getSharedPrefFile(account),
         		Context.MODE_PRIVATE);
         final Editor editor = prefs.edit();
         editor.putLong(BACKOFF, backoff);
-        editor.commit();
+        editor.apply();
```

Comment:
I thought `apply()` was async (vs) `commit()` was sync, but I read more about it and it appears that `apply()` is probably more efficient since it's atomic to an in-memory cache, but async to disk. 👍🏻